demos/chocolateyfest2018.ps1

```diff
@@ -24,18 +25,43 @@ function executeScript {
 #--- Setting up Windows ---
 executeScript "FileExplorerSettings.ps1";
 executeScript "SystemConfiguration.ps1";
-executeScript "CommonDevTools.ps1";
 executeScript "RemoveDefaultApps.ps1";
+executeScript "CommonDevTools.ps1";
+executeScript "Browsers.ps1";
+
+executeScript "HyperV.ps1";
+RefreshEnv
 executeScript "WSL.ps1";
-executeScript "VirtualizationTools.ps1";
+RefreshEnv
+executeScript "Docker.ps1";
+
+choco install azure-cli
+choco install azurepowershell
```

Comment:
Instead, can we do `Install-Module Az` here? Trying to promote the new module they're shipping now, it's PS Core compatible and uses better cmdlet names (with aliases back to the old)